src/layout/header/styles.module.scss

```diff
@@ -1,3 +1,102 @@
-.root {
+@use '../../styles/global' as *;
```

Comment:
The other benefit of keeping styles related to the header in subcomponents of the header is that it simplifies stylesheets like this one.